mbed_cloud_client_user_config.h

```diff
@@ -34,7 +34,7 @@
 #endif
 
 /* set flag to enable update support in mbed Cloud client */
-#define MBED_CLOUD_CLIENT_SUPPORT_UPDATE
+#undef MBED_CLOUD_CLIENT_SUPPORT_UPDATE
```

Comment:
Create a new mbed_cloud_client_user_config_legacy.h file for legacy targets.

Reply:
Done